src/components/select/select.js

```diff
@@ -175,15 +186,14 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $interpolate,
         }
       }
 
-
       var originalRender = ngModelCtrl.$render;
-      ngModelCtrl.$render = function() {
+      ngModelCtrl.$render = function () {
```

Comment:
There actually shouldn't be a space between `function` and the parens (here and elsewhere)